package.json

```diff
@@ -46,9 +52,25 @@
     "shx": "^0.2.2",
     "sinon": "^1.17.6",
     "ts-node": "^3.3.0",
-    "typescript": "~2.6.1"
+    "typescript": "~2.6.1",
+    "husky": "2.2.0",
+    "lint-staged": "8.1.6",
+    "prettier": "1.17.0"
   },
   "dependencies": {
     "tslib": "~1.8.1"
+  },
+  "lint-staged": {
+    "*.{ts,tsx}": [
```

Comment:
should also lint css

Reply:
would require a different linter etc though right?

Reply:
I think it picks it up automatically